spec/integration/puppet_x/puppetlabs/iis/powershell_manager_spec.rb

```diff
@@ -174,6 +193,9 @@ def close_stream(stream, style = :inprocess)
       end
 
       it "should create a new PowerShell manager host if the output stream handle is closed" do
+        # currently skipped as it can trigger an internal Ruby thread clean-up race
+        # its unknown why this test fails, but not the identical test against @stderr
+        skip('This test can cause intermittent segfaults in Ruby with w32_reset_event invalid handle')
```

Comment:
Try wrapping it like this:
skip "This test can cause intermittent segfaults in Ruby with w32_reset_event invalid handle" do
...
end

Reply:
This bit shouldn't be a problem - this is regular rspec and same code works fine in other modules.